datasets/mk/elected_officials/crawler.py

```diff
@@ -0,0 +1,74 @@
+from typing import Dict
+import requests
+
+from zavod import Context
+from zavod import helpers as h
+from zavod.logic.pep import categorise
+
+IGNORE_COLUMNS = [
+    "formType",
+    "subInstitution",
+]
+
+
+def crawl_row(context: Context, row: Dict[str, str]):
+    person = context.make("Person")
+
+    person.id = context.make_slug(row.pop("id"))
```

Comment:
I'm a bit uncomfortable using these IDs as person IDs - it's probably safe enough but since these seem to refer to form submissions, how about making the ID the compose of first name, last name, position, institiution? You can just pass all these as arguments to context.make_id and it'll slugify and hash them.

Reply:
Very good call!

I've only checked if the id was unique, and not if it would properly identify a given individual such as:

|id                |name   |lastName  |workingPosition |institution             |dateNaming |dateTermination |
|:-----------------|:------|:---------|:---------------|:-----------------------|:----------|:---------------|
|mk-officials-4541 |ЖАНЕТА |ЧАУШЕВСКА |градоначалник   |ОПШТИНА ГРАДСКО Градско |NA         |2017-11-03      |
|mk-officials-7293 |ЖАНЕТА |ЧАУШЕВСКА |градоначалник   |ОПШТИНА ГРАДСКО Градско |2013-04-03 |NA              |

Done in https://github.com/opensanctions/opensanctions/pull/535/commits/3d64a46d05e4ccab7cf04e830f132376c6f4856c